Tests/Marketplace/search_and_install_packs.py

```diff
@@ -391,11 +391,15 @@ def get_latest_version_from_bucket(pack_id: str, production_bucket: Bucket) -> s
     """
     pack_bucket_path = os.path.join(GCPConfig.STORAGE_BASE_PATH, pack_id)
     # Adding the '/' in the end of the prefix to search for the exact pack id
```

Comment:
```suggestion
    # Adding the '/' in the end of the prefix to search for the exact pack id
```
move one line down please